modules/oe_theme_helper/translations/oe_theme_helper-bg.po

```diff
@@ -339,3 +339,9 @@ msgstr "Скриване на координатите за връзка"
 
 msgid "Leadership and organisation"
 msgstr "Ръководство и организация"
+
+msgid "Page contents"
```

Comment:
`msgid "Page contents"` has been added in https://github.com/openeuropa/oe_theme/pull/874/files

Reply:
Removed